### ISSUE TITLE
[BUG] pex13 and voidui13 doesn't work

### ISSUE THREAD

**Issue by kuind** [Mahmud0808/Iconify#151]
### Describe the bug

pex13 and voidui13 doesn't work

### Expected behavior

Unable to open the software after installation

### Screenshots

![Screenshot_20230513-155102_Iconify](https://github.com/Mahmud0808/Iconify/assets/74531773/910a4012-6d3c-4eea-b733-870121319f7a)
![Screenshot_20230513-154542_MT管理器](https://github.com/Mahmud0808/Iconify/assets/74531773/04d64a91-b975-4d7d-be00-8fd716734a33)

### Device information

Iconify bug report 13 5月 2023 15:39:57
version: 6.1.1 (9)
build.brand: Xiaomi
build.device:  
build.display: TQ2A.230505.002
build.fingerprint: Redmi/alioth/alioth:12/SKQ1.211006.001/V13.0.9.0.SKHCNXM:user/release-keys
build.hardware: qcom
build.id: TQ2A.230505.002
build.manufacturer: Xiaomi
build.model: M2012K11AC
build.product: aosp_alioth
build.type: user
version.codename: REL
version.release: 13
version.sdk_int: 33
/

### Additional context

Iconify bug report 13 5月 2023 15:39:57
version: 6.1.1 (9)
build.brand: Xiaomi
build.device: alioth
build.display: TQ2A.230505.002
build.fingerprint: Redmi/alioth/alioth:12/SKQ1.211006.001/V13.0.9.0.SKHCNXM:user/release-keys
build.hardware: qcom
build.id: TQ2A.230505.002
build.manufacturer: Xiaomi
build.model: M2012K11AC
build.product: aosp_alioth
build.type: user
version.codename: REL
version.release: 13
version.sdk_int: 33

error: Failed to zip align NBHideKBButton-unsigned.apk

OverlayCompiler - ZipAlign:
	Unable to open '/storage/emulated/0/.iconify/overlays/unsigned_unaligned/NBHideKBButton-unsigned-unaligned.apk' as zip archive

**Comment by Mahmud0808** [comment]
Did you try debug build? Use debug build with matlog or any log checker app. Generate the logs and send it here as a file.

**Comment by kuind** [comment]
/[Log_2023-05-15_06-06-19.txt](https://wwxo.lanzouf.com/ipoQK0w7jt9c)

**Comment by Mahmud0808** [comment]
> /[Log_2023-05-15_06-06-19.txt](https://wwxo.lanzouf.com/ipoQK0w7jt9c)

What's the error this time? Same as before?

**Comment by kuind** [comment]
Yes, still the same,open the App and follow the steps, but the last step cannot be completed, prompting "Oops Did Not Expect That
Logs have been saved in the documents folder.
Let's Go", tried many versions of this.[2023-05-15_16.23.zip](https://wwxo.lanzouf.com/iXJFX0w8sj9c)

**Comment by Mahmud0808** [comment]
I don't see any clear error in log. Probably it's from ZipAlign jni library. Can't even reproduce the issue on my device.

**Comment by kuind** [comment]
[Is this log？ Log_2023-05-15_18-28-40.txt](https://wwxo.lanzouf.com/igOct0w96fkd)

**Comment by Mahmud0808** [comment]
> [Is this log？ Log_2023-05-15_18-28-40.txt](https://wwxo.lanzouf.com/igOct0w96fkd)

Thanks for providing logs. I will check if there is any possible fix.

**Comment by shestak1996** [comment]
Hello. I have the same error.
[iconify_logcat_14-11-23_01_39_39.txt](https://github.com/Mahmud0808/Iconify/files/13342889/iconify_logcat_14-11-23_01_39_39.txt)

**Comment by Aveffbane** [comment]
> > [Is this log？ Log_2023-05-15_18-28-40.txt](https://wwxo.lanzouf.com/igOct0w96fkd)
> 
> Thanks for providing logs. I will check if there is any possible fix.

Hello. I have the same error.
[iconify_logcat_14-11-23_01_39_39.txt](https://github.com/Mahmud0808/Iconify/files/13342894/iconify_logcat_14-11-23_01_39_39.txt)

**Comment by Mahmud0808** [comment]
> Hello. I have the same error. [iconify_logcat_14-11-23_01_39_39.txt](https://github.com/Mahmud0808/Iconify/files/13342894/iconify_logcat_14-11-23_01_39_39.txt)

Are you using MIUI? This error happens only for modded framework users.
